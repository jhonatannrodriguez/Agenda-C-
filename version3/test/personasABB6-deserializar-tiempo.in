deserializarArbolPersonasTiempo 10000 2

deserializarArbolPersonasTiempo 16383 2

deserializarArbolPersonasTiempo 16381 2

Fin