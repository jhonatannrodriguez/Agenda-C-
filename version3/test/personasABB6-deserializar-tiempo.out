1>La amplitud del árbol es: 4096.
La cantidad de personas en el árbol es: 10000.
Bien.
2>La amplitud del árbol es: 8192.
La cantidad de personas en el árbol es: 16383.
Bien.
3>La amplitud del árbol es: 8190.
La cantidad de personas en el árbol es: 16381.
Bien.
4>Fin.
